msal/src/main/java/com/microsoft/identity/client/MultipleAccountPublicClientApplication.java

```diff
@@ -366,7 +366,6 @@ private void removeAccountInternal(@Nullable final IAccount account,
         // The broker strips these properties out of this object to hit the cache
         // Refactor this out...
         final AccountRecord requestAccountRecord = new AccountRecord();
-        requestAccountRecord.setEnvironment(multiTenantAccount.getEnvironment());
```

Comment:
Why remove this line?

Reply:
We don't want to pass in specific environment to removeAccount api, as we want to remove accounts in all environment.

Reply:
I see so the environment here is NOT on the API but rather on an Account object that is then passed to the remove account api and the api will figure out the environment from there...is that right?

Reply:
Yes, that's right.